File: samples/Marketplace/Marketplace.Web/app/services/StatusTypeService.ts
// Copyright 2017 (c) [Denis Da Silva]. All rights reserved.
// See License.txt in the project root for license information.

/// <reference path="../_References.ts" />

module App.Services {
    "use strict";

    export class StatusTypeService extends UpdateRestService<Models.StatusType, Queries.CategoryQuery> {
        static $inject = ["$http", "$q", "Application"];

        constructor(public $http: ng.IHttpService, $q: ng.IQService, public Application: IApplication) {
            super($http, $q, Application);

            this.ApiController = "StatusType";
        }
    }
}

File: samples/Marketplace/Marketplace.Logic/Services/Configuration/StatusTypeService.cs
// Copyright 2017 (c) [Denis Da Silva]. All rights reserved.
// See License.txt in the project root for license information.

using Marketplace.Data;
using Marketplace.Domain.Models.Configuration;
using Marketplace.Domain.Security;
using Marketplace.Logic.Contracts.Configuration;

namespace Marketplace.Logic.Services.Configuration
{
    public sealed class StatusTypeService : RepositoryService<StatusType>, IStatusTypeService
    {
        public StatusTypeService(IDataContext context, SecurityContext security) : base(context, security)
        {
        }
    }
}
